scripts/zones/East_Ronfaure/npcs/Geomantic_Reservoir.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: East Ronfaure
+-- NPC: Geomantic Reservoir
+-- Unlocks Geo-Poison
+-- !pos 378.875 -39.304 58.313
+-----------------------------------
+local ID = require("scripts/zones/East_Ronfaure/IDs")
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:hasSpell(769) then -- Player has Indi-Poison
+        player:startEvent(15000, 277, 3, 380619, -39102, 53620, 163, 539943, 0)
+    else
+        player:PrintToPlayer("You are assaulted by an uncanny sensation.", 0xD)
```

Comment:
We don't use `PrintToPlayer` for any messages sent to the client, since they're all baked into the client.
See here:
https://github.com/LandSandBoat/server/blob/base/scripts/quests/hiddenQuests/Trust_Ingrid.lua#L41
`player:messageSpecial(easternAdoulinID.text.YOU_LEARNED_TRUST, ...`

You can use POLUtils to scrape your installed client and it'll show you all of the text IDs for each zone:
https://github.com/Windower/POLUtils

These scrapes ARE A YEAR OLD DO NOT USE! - but they show the output of POLUtils, and the exact text you need to use for our update tooling to continue to work:
https://github.com/zach2good/UpdateExtractor/blob/versions/2021-july/out/scripts/zones/East_Ronfaure/Text.lua.raw

Like I say: THESE ARE OUT OF DATE DO NOT USE - but this is the text in this area:
```
11958 : % learns %!
11960 : You are assaulted by an uncanny sensation.
11961 : The arcane energies begin to course within your veins.
11962 : You feel a mystical warmth welling up inside you!
11963 : The heat seems to have drained from your %.
11964 : Something's not quite right. Head to %, where Arciela awaits you.
```